### DISCUSSION TITLE
🌐 Help translate NetBox! Language coordinators needed.

### DISCUSSION THREAD

**Discussion by jeffgdotorg** [netbox-community/netbox#16744] [Announcements]
**Attention all multilingual NetBox fans!**

We're pushing to complete high-quality translations of NetBox for more languages. It's a lot of work, and we're lucky to have a great bunch of volunteers already signed up to help out in 9 languages. At the moment we have a particular need for language coordinators to oversee work on the following languages:
- Chinese 🇨🇳 
- German 🇩🇪 
- Turkish 🇹🇷 
- Ukrainian 🇺🇦 

If you're proficient in one or more of those languages and want to help out in this capacity, please reply to this discussion and one of the maintainers will get in touch with more information.

Of course we're always on the lookout for translators and reviewers. We currently have translation teams for NetBox in Chinese, French, German, Japanese, Portuguese, Russian, Spanish, Turkish, and Ukrainian. If you're not up for the coordinator gig but would like to join one of those teams, do raise your hand too.

**Comment by p3psi-boo** [comment]
I would like to volunteer as the Chinese language coordinator for the NetBox translation efforts. Please provide further details on how I can contribute effectively.

**Reply by jeffgdotorg** [reply]
Excellent, thanks for stepping up!

You'll need to [request to join](https://app.transifex.com/signup/open-source/?join_org=netbox-community&join_project=netbox) the `netbox-community` org on Transifex as a first step. Once you've done that, reply here with your Transifex username and I'll add you to the project.

**Reply by p3psi-boo** [reply]
Transifex username is `bujinpeng`.

**Reply by jeffgdotorg** [reply]
Invitation sent!

**Comment by luojiyin1987** [comment]
I want to translate it to Chinese 🇨🇳 .   [about me](https://www.freecodecamp.org/chinese/news/author/luojiyin/)

**Reply by jeffgdotorg** [reply]
Great, thanks for raising your hand!

You'll need to [request to join](https://app.transifex.com/signup/open-source/?join_org=netbox-community&join_project=netbox) the `netbox-community` org on Transifex as a first step. Once you've done that, reply here with your Transifex username and I'll add you to the project.

**Reply by luojiyin1987** [reply]
My  Transifex username is  luojiyin

**Reply by jeffgdotorg** [reply]
I see you are already a translator. Would you like to be a reviewer as well?

**Reply by luojiyin1987** [reply]
ok，I  can use my free time.

**Reply by jeffgdotorg** [reply]
Okay, you are now a reviewer. Thank you!

**Comment by Songxwn** [comment]
I need to join the reviewer, and our small team has built a set of Chinese translations available.

[Blog](https://songxwn.com/netbox4-CN/)

**Reply by jeffgdotorg** [reply]
We always need reviewers, glad to have you!

You'll need to [request to join](https://app.transifex.com/signup/open-source/?join_org=netbox-community&join_project=netbox) the `netbox-community` org on Transifex as a first step. Once you've done that, reply here with your Transifex username and I'll add you to the project.

**Reply by jeffgdotorg** [reply]
I just invited you as a reviewer. Thanks for volunteering!

**Reply by Songxwn** [reply]
songxwn

**Comment by click0** [comment]
So far I am the leader in translation into Ukrainian.
But I would like @jeremystretch to enable a dictionary for this language.

**Reply by click0** [reply]
I prefer the Crowdin translation service for convenience.

**Reply by jeffgdotorg** [reply]
We've made a big investment of time and effort in Transifex and adapted our dev process to accommodate it. Adding another platform to our tooling isn't something we're prepared to consider at the moment.

Please message me on NetDev Slack or via Transifex and I'll help get you going with a dictionary.

**Reply by click0** [reply]
> Please message me on NetDev Slack or 

Slack is not popular in Eastern Europe.

> via Transifex and I'll help get you going with a dictionary.

In the Transifex service, simple translators cannot see the list of Netbox team members.
I indirectly see only Jeremy Stretch @jeremystretch, who says there "oversees the NetBox project"

I found you there and sent you a private message

**Reply by jeffgdotorg** [reply]
I created an empty glossary assigned to the NetBox project. It should now be accessible from all languages.

**Comment by Kampfschwein99** [comment]
Hey, I would like to help out with the German Translation. I´m a German native speaker.

**Reply by jeffgdotorg** [reply]
Ausgezeichnet!

You'll need to [request to join](https://app.transifex.com/signup/open-source/?join_org=netbox-community&join_project=netbox) the `netbox-community` org on Transifex as a first step. Once you've done that, reply here with your Transifex username and I'll add you to the project.

**Reply by Kampfschwein99** [reply]
My username on Transifex is Kampfschwein99

**Reply by jeffgdotorg** [reply]
I see you're already listed as a translator. Please let me know if you would like to be a Reviewer or Coordinator.

**Reply by chbally** [reply]
@Kampfschwein99 I openend under the German Translation Discussion a[ new threat](https://github.com/netbox-community/netbox/discussions/16171#discussioncomment-9913222) maybe you want to use that for coordination.

**Reply by click0** [reply]
> @Kampfschwein99 I openend under the German Translation Discussion a[ new threat](https://github.com/netbox-community/netbox/discussions/16171#discussioncomment-9913222) maybe you want to use that for coordination.

@jeffgdotorg 
Please add the Translation category to the [Discussion](https://github.com/netbox-community/netbox/discussions).

**Reply by jeffgdotorg** [reply]
Done. Good catch, thanks.

**Reply by jeffgdotorg** [reply]
Just realized what you meant. I've created the [I18N](https://github.com/netbox-community/netbox/discussions/categories/i18n) discussion category.

**Comment by bastianleicht** [comment]
Hey, I would like to help with the German Translation. I'm also a German native speaker.

My username on Transifex is: bastianleicht

**Reply by jeffgdotorg** [reply]
Great, thanks for your interest! I'll get you added as soon as I've been granted the required permissions.

**Reply by jeffgdotorg** [reply]
I see you're already listed as a translator. Please let me know if you would like to be a Reviewer or Coordinator.

**Reply by bastianleicht** [reply]
I would like to review. There is not a lot left to do.

**Reply by chbally** [reply]
@bastianleicht   I openend under the German Translation Discussion a[ new threat](https://github.com/netbox-community/netbox/discussions/16171#discussioncomment-9913222) maybe you want to use that for coordination.

**Reply by jeffgdotorg** [reply]
@bastianleicht you are now a reviewer. Vielen Dank!

**Comment by rizlas** [comment]
Add me for Italian. Same username as in github.

**Reply by jeffgdotorg** [reply]
Grazie! I'll add you once I've been granted the necessary permissions.

**Reply by jeffgdotorg** [reply]
Is your Transifex profile set to private? I am unable to add you by that username.
<img width="663" alt="image" src="https://github.com/netbox-community/netbox/assets/1209268/d4ff7e97-aa04-4baa-b937-af44a19f6518">

**Reply by rizlas** [reply]
Ok, try again. I had privacy enabled

**Reply by jeffgdotorg** [reply]
@rizlas thank you. I invited you as a translator. If you would like to be a reviewer or coordinator, please let me know.

**Reply by rizlas** [reply]
Hi @jeffgdotorg , please add me and @zvfvrv as reviewer so we can check the auto translation.

**Reply by rizlas** [reply]
Any update on this? With the latest release there are already wrong tranlsations.

**Comment by zvfvrv** [comment]
Hello, I would like to help with the Italian translation.

**Reply by jeffgdotorg** [reply]
Grazie! I'll add you once I've been granted the necessary permissions.

**Reply by jeffgdotorg** [reply]
What is your Transifex username? I tried inviting you using your GH ID but got the same result as with @rizlas above.

**Reply by zvfvrv** [reply]
Hi, yes just updated the profile

**Reply by jeffgdotorg** [reply]
@zvfvrv thank you. I invited you as a translator. If you would like to be a reviewer or coordinator, please let me know.

**Comment by Houbsi** [comment]
I have sent a request to join for apparently all the languages i speak. (Deutsch, English, Francais and dutch)

**Reply by jeffgdotorg** [reply]
@Houbsi thanks! You're now a translator for Français and Deutsch, and invited to be a translator on the brand-new Nederlands team. Please let me know if you would like to be a reviewer or coordinator for any of those teams.

**Reply by Houbsi** [reply]
@jeffgdotorg 
Dutch is the language i'm least good at. 
German - native
English - 2nd and most talked language.
French - good'ish
Dutch - existing but not good enought to be coordinator, coordinating, i'm not sure :D

**Comment by s-teffen** [comment]
I can support with reviewing the german translation. My transfex username is s_teffen.

**Reply by jeffgdotorg** [reply]
Thanks! I've invited you to the team.

**Comment by Simplicity-Poland** [comment]
Right after receiving an email that NetBox is now being translated to Polish, I've joined the project on Transifex. Please accept my join request.

**Reply by jeffgdotorg** [reply]
Thanks! I've accepted your join request and you're now a Translator. If you would like to be promoted to Reviewer or Coordinator, please let me know.

**Comment by czarnian** [comment]
Hi, I received a notification that Czech is being added, so please, accept my join request. I am eager to start :-)

**Comment by click0** [comment]
Please add Export and Import of project/files for translation in YAML format, as poeditor.com did.
If this is not in the admin settings, then forward this request to the translation service transifex.com.

Thanks.

**Comment by Wythetells** [comment]
I want to translate it to Chinese 🇨🇳 .my transifex's name is jywyx2.
